jupyterhub_singleuser_profiles_ui/src/ImageForm/components/ImageForm.js

```diff
@@ -55,17 +55,12 @@ class ImageForm extends React.Component {
         console.log("Sent image")
     }
 
-    DropdownValue() {
-        if (this.state.selectedValue !== null && this.state.selectedValue !== '') {
-            return this.state.selectedValue
+    checkListLength() {
+        if(this.state.imageList.length !== 0) {
```

Comment:
Can you explain what is happening here?

Reply:
There is no "default" image, so I just check if there is atleast one image in the list of images and set the first one as the default value before the user chooses any. Otherwise I put in "No images available". I will rewrite this so it does not run every render though.

Reply:
it is only neccessary if the person is spawning a notebook and he does not have a user configMap yet.